frontend/src/pages/pipelines/global/experiments/artifacts/ArtifactOverviewDetails.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+
+import {
+  Flex,
+  FlexItem,
+  Stack,
+  Title,
+  DescriptionList,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  DescriptionListDescription,
+} from '@patternfly/react-core';
+
+import { Artifact, Value } from '~/third_party/mlmd';
+import { usePipelinesUiRoute } from '~/concepts/pipelines/context/usePipelinesUIRoute';
```

Comment:
`usePipelineUIRoute` -> `usePipelineUiRoute`?

Reply:
Yes, I mistyped this. Updating now.